src/middlewared/middlewared/plugins/cron.py

```diff
@@ -41,14 +55,38 @@ def cron_extend(self, data):
                     'Specified user does not exist'
                 )
 
+        command = data.get('command')
+        if not command:
+            verrors.add(
+                f'{schema}.command',
+                'Please specify a command for cronjob task.'
+            )
+        else:
+            crontab_cmd = (await self.construct_cron_command(
+                data['schedule'], user, command, data['stdout'], data['stderr']
+            ))[6:]
+            command = crontab_cmd.pop(1)
+
+            # cron(8) when reads an entry from a crontab, it keeps a buffer of 1000 characters and anything more then
+            # that is truncated. So we validate that the user supplied command is not of more then 1000 characters.
+            allowed_length = 1000 - len(' '.join(crontab_cmd))
+
+            if len(command) > allowed_length:
+                verrors.add(
+                    f'{schema}.command',
+                    f'Command must be less then or equal to {allowed_length} characters. '
+                    'It should be noted that in the command provided, all newline characters are removed and '
```

Comment:
```suggestion
                    'Any newline or "%" characters in the command are automatically removed or escaped with '
```

Reply:
@Mrt134 newline characters are not escaped with a backslash ;) How about we make that explicit here ? Or here implies that both cases can happen ( removed/escaped ) which is of course not the case ?

Reply:
@Mrt134 ping

Reply:
@freenas/docs-team ping

Reply:
There's a couple different ways. It's currently suggested as an understood implication that first item correlates to first item, but you can add `, respectively` to the end of the sentence to clarify or rewrite to break up the paired pairs:
"Any newline or "%" characters in the command are automatically removed or escaped with backslash, respectively."
or
"Newline characters are automatically removed and "%" characters escaped with a backslash."

Reply:
Perfect, thank you :) I'll have this covered.

Reply:
No problem! Sorry it took a bit to get back to you.